src/components/Item.js

```diff
@@ -0,0 +1,73 @@
+import React, { useEffect } from 'react';
+import '../components/AddItemForm.css';
+import { updateDoc, serverTimestamp, doc } from 'firebase/firestore';
+import { useState } from 'react/cjs/react.development';
```

Comment:
@Mandihamza @timamero I think this is the cause of the error. 

```suggestion
import { useState } from 'react;
```

Reply:
After making the suggested change the deploy preview is running without error.  Thank you @jonathanyeong !